webclient/src/components/Loading.js

```diff
@@ -2,19 +2,20 @@ import React from 'react';
 
 function Loading() {
   return (
-    <div className="preloader-wrapper big active">
-      <div className="spinner-layer spinner-blue">
-        <div className="circle-clipper left">
-          <div className="circle"></div>
-        </div>
-        <div className="gap-patch">
-          <div className="circle"></div>
-        </div>
-        <div className="circle-clipper right">
-          <div className="circle"></div>
-        </div>
-      </div>
-    </div>
+    // <div className="preloader-wrapper big active">
```

Comment:
Is this commented section needed?

Reply:
Might be for future use.. it is the loading muter2